production/scrapers/nevada.R

```diff
@@ -17,7 +17,7 @@ nevada_check_date <- function(url, date = Sys.Date()){
         unlist() %>%
         {.[str_detect(., "20")]} %>% # look for year 20xx
         lubridate::mdy()
-
+    
```

Comment:
Could you limit the changes in this PR to just those that affect the code operation? So here, we've shifted from no space to a tabbed line in. I'd prefer to preserve the original no tabbed in, and then address any broader style things separately.